src/test/suite/editors/playgroundSelectedCodeActionProvider.test.ts

```diff
@@ -195,7 +195,7 @@ suite('Playground Selected CodeAction Provider Test Suite', function () {
       }
     });
 
-    test('exports to java and includes builders', async () => {
+    test('exports to java and includes builders and import statements', async () => {
```

Comment:
Can we have actually two tests, one when we only include builders and another one for both builders and imports?